helm/velero-app/Chart.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: v1
+apiVersion: v2
 name: velero-app
```

Comment:
From https://intranet.giantswarm.io/docs/dev-and-releng/app-developer-processes/adding_app_to_appcatalog/

> However, it is no longer required nor advisable to name your app and helm chart (step 2 & 3 below) with the -app suffix.

@ubergesundheit , this applies here, right? So better to name it `velero`?

Reply:
if its a default app installed as part of a release, the `-app` suffix is still required

Reply:
removed app suffix